### DISCUSSION TITLE
Vault unlock failed Error- unable to get local issuer certificate

### DISCUSSION THREAD

**Discussion by Mayapep69** [buttercup/buttercup-desktop#1340] [Q&A] [answered]
I am getting the below error on a WIN 11 system when I am attempting to open my vault for the desktop application. I have never gotten this before and am at a loss on how to resolve it. 

Vault unlock failed: Error invoking remote method 'unlock-source': Layerr: Failed unlocking vault source: Failed unlocking source: request to https://content.dropboxapi.com/2/files/download failed, reason: unable to get local issuer certificate.

**Comment by perry-mitchell** [comment, marked as answer]
This looks to be machine specific. Obviously neither Buttercup nor electron/node should interrupt ssl verification, so it's most likely either intermittent or something to do with your machine or network.

See this: https://developer.genesys.cloud/forum/t/axioserror-unable-to-get-local-issuer-certificate-error-while-using-platform-api-js-sdk/21106

Seems to be somewhat common at least.
